pillar/journal-public.sls

```diff
@@ -66,6 +66,11 @@ journal:
 
     subject_rewrites: []
 
+    feature_rds: false
+    rds_articles:
+        '30274':
+            display: https://hub.stenci.la/elife/30274/main/
```

Comment:
Adding here rather than in just in `preview` to avoid duplication when adding `prod`.